### ISSUE TITLE
Tentative 1.5.0 Release

### ISSUE THREAD

**Issue by kbhomes** [radiant-player/radiant-player-mac#388]
@radiant-player/radiant-player-mac: I'm planning on releasing the new version 1.5.0 likely tomorrow, because it fixes the major problems including media keys (and other related bugs), CSS fixes, Last.fm support, and a far more robust update system with Sparkle. I think it would be best to get this out sooner than later, but if you have any concerns or objections let me know. :+1:

**Comment by chrismou** [comment]
:+1: I'm spending the day running off the master branch, so I'll let you know if I come across any issues

**Comment by chrismou** [comment]
Strangely, last.fm integration seems to have stopped working again on master, even with the fix in, though the binary from my bugfix branch still seems to be scrobbling OK.

Can anyone else confirm they're seeing this issue or is it just me? The Xcode console doesn't look to be reporting any errors, but nothing's showing up on my profile.

**Comment by chrismou** [comment]
OK, it looks like the call to the last.fm api isn't actually happening at all on the master branch. Looking at the requests through a web debugging proxy, a working build calls ws.audioscrobbler.com multiple times (see below), one of which updates the "now playing" track. On that master branch, those calls aren't made at all.

Am sifting through commits trying to puzzle out what's changed between the versions. If anyone has any insight, let me know.

As a side note - I've also logged #390 as I've noticed the multiple API calls seems to be unloving tracks, which I'm guessing isn't by design ;-)
